Makefile

```diff
@@ -3,8 +3,9 @@ JAIL_NIC?=vtnet0
 JAIL_IP?=172.16.0
 JAIL_NET?=16
 MYPYPATH = $(shell pwd)/.travis/mypy-stubs
-PYTHON ?= python3.6
 
+LATEST_PKG_PYTHON_VERSION != pkg info -g 'python3*' | cut -d'-' -f1 | sed 's/^python//' | sort -n | tail -n1 | sed -r 's/^([0-9])([0-9]+)/\1.\2/'
+PYTHON ?= python${LATEST_PKG_PYTHON_VERSION}
 pyver= ${PYTHON:S/^python//:S/.//:C/\([0-9]+\)/\1/}
```

Comment:
given that we already have `LATEST_PYTHON_VERSION` we can use this directly here